### ISSUE TITLE
Installation: onnxruntime import error

### ISSUE THREAD

**Issue by vs74** [ivadomed/ivadomed#504]
Hello, 

I'm new to ivaodmed and trying to experiment with it, so I was following the tutorials as mentioned in[ RTD](https://ivadomed.org/en/latest/tutorials/one_class_segmentation_2d_unet.html). 

When I ran `ivadomed_download_data -d data_example_spinegeneric`, i get the following error log

```
ivadomed_env) (base) vasudevs-MacBook-Pro:ivadomed vasudevsharma$ ivadomed_download_data -d data_example_spinegeneric

/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/site-packages/onnxruntime/capi/_pybind_state.py:14: UserWarning: Cannot load onnxruntime.capi. Error: 'dlopen(/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/site-packages/onnxruntime/capi/onnxruntime_pybind11_state.so, 2): Library not loaded: /usr/local/opt/libomp/lib/libomp.dylib
  Referenced from: /Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/site-packages/onnxruntime/capi/onnxruntime_pybind11_state.so
  Reason: image not found'.
  warnings.warn("Cannot load onnxruntime.capi. Error: '{0}'.".format(str(e)))
Traceback (most recent call last):
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/bin/ivadomed_download_data", line 33, in <module>
    sys.exit(load_entry_point('ivadomed', 'console_scripts', 'ivadomed_download_data')())
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/bin/ivadomed_download_data", line 25, in importlib_load_entry_point
    return next(matches).load()
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/site-packages/importlib_metadata/__init__.py", line 105, in load
    module = import_module(match.group('module'))
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed/__init__.py", line 1, in <module>
    from .utils import __version__, __ivadomed_dir__
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed/utils.py", line 11, in <module>
    import onnxruntime
  File "/Users/vasudevsharma/VScode/Projects/ivadomed/ivadomed_env/lib/python3.6/site-packages/onnxruntime/__init__.py", line 13, in <module>
    from onnxruntime.capi._pybind_state import get_all_providers, get_available_providers, get_device, set_seed, \
ImportError: cannot import name 'get_all_providers'
```

To my understanding, this is because of improper install of ivadomed, but I properly installed it as mentioned in the [installation instruction](https://ivadomed.org/en/latest/installation.html).

In brief, I did the following steps:-
1. Fork the repo, which is in sync with the latest master branch of ivadomed
2. Made a virtual environment as mentioned in the installation section of RTD
3. In the ivadomed directory, did `pip install - e .` to install the development version of it.

Any help regarding why I'm encountering **onnxruntime import error** would be greatly appreciated.

**Comment by andreanne-lemay** [comment]
Hi Vasudev,
Thank you for opening this issue. A similar problem has been reported by the SCT regarding this issue with ivadomed: https://github.com/neuropoly/spinalcordtoolbox/issues/2930.
I think forcing the version to `1.4.0` in onnxruntime should solve the issue. I'll open a PR to resolve this issue.

**Comment by andreanne-lemay** [comment]
@vs74 Let me know if this fixes it for you #505

**Comment by vs74** [comment]
Thanks forcing `onnxruntime `version to be `1.4.0` fixes the issue. Thanks for quick turnaround.
